menpo/image/base.py

```diff
@@ -322,6 +322,53 @@ def init_from_rolled_channels(cls, pixels):
         """
         return cls(np.rollaxis(pixels, -1))
 
+    @classmethod
+    def init_from_pointcloud(cls, pointcloud, group=None, boundary=0,
+                             n_channels=1, fill=0, dtype=np.float):
+        r"""
+        Create an Image that is big enough to contain the given pointcloud.
+        The pointcloud will be translated to the origin and then translated
+        according to its bounds in order to fit inside the new image.
+        An optional boundary can be provided in order to increase the space
+        around the boundary of the pointcloud. The boundary will be added
+        to *all sides of the image* and so a boundary of 5 provides 10 pixels
+        of boundary total for each dimension.
+
+        Parameters
+        ----------
+        pointcloud : :map:`PointCloud`
+            Pointcloud to place inside the newly created image.
+        group : `str`, optional
+            If ``None``, the pointcloud will only be used to create the image.
+            If a `str` then the pointcloud will be attached as a landmark
+            group to the image, with the given string as key.
```

Comment:
landmark attachment very nice addition, definitely useful for most calling cases for this constructor.

This is perhaps another case where it would be nice to have access to the underlying transform that was used as well, a-la
https://github.com/menpo/menpo/blob/master/menpo/image/base.py#L1817

Do we do the same and add a `return_transform=False` kwarg? Little gross as it's on a pseudo-constructor, but potentially useful and makes it consistent with other image APIs.